### PR TITLE
Initial Docs v1.0

### PR DESCRIPTION
**Pull request by y0014984**
# Features

- Articles about Explore, Users, Communities, Events, Markdown and Slotlists
- Additional Articles about Team, FAQ, Support, Feedback, Contribution, Roadmap, Changelog and Tech Stack
- Images and Videos
- Double Nav Bar
- Social Links
- Localisation EN, DE
  - https://docs.arma.events/en/
  - https://docs.arma.events/de/
- Footer (Copyright etc.)
- "Edit on Github" links
- Search Bar
- Logo and title image
- Last Updated Info
- Image Zoom with medium-zoom (https://github.com/vuejs/vitepress/issues/854)
  - `npm i medium-zoom`
- Favicon and robots.txt
- Lazy Loading (with WEBM and WEBP media files)

### CODE REVIEW

**Comment by gelir95** [comment]
To enable a better expandebility and to prepare for a multilingual documentation in the future I would suggest following folder structure:

docs/ 
├─ de/       │ 
                    ├─ foo.md  │
                                          ├─ guides/ │
                                                                ├─ foo.md
foo.md         
├─ guides/│
                     ├─ foo.md

Documents inside the "de" section should be present to encourage translation. 
The "guide" section is best used for the documents created in this PR. Im not sure if the baselevel document is required with this structure.

For reference look at the vitepress site: 
https://vitepress.dev/guide/what-is-vitepress

Additionally to enable multilingual support the config.ts file needs to be edited according to the vitepress documentation:
https://vitepress.dev/guide/i18n

**Comment by gelir95** [comment]
The images folder should use subfolders for each md document.

File: user.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Keep in mind that the community will be deleted if you are the only/last member. If there are other members but you are the only administrator of this community then you need to transfer your admin rights to another member. See [communities](communities.md "Communities") for more information about community management.
+
+![Sessions](images/user-profile-communities-overview.png "Sessions")
+
+### Sessions
+
+Session management is a security feature. It lists every active browser session on your devices. Feel free to close/revoke all sessions you no longer need or all sessions at once. A revoked session means that you need to login again on that specific device/browser combination.
+
+![Sessions](images/games-and-dlcs.png "Sessions")
```
Comment (gelir95) [inline user.md]: Picture links to games-and-dlcs  instead of sessions.

**Comment by y0014984** [comment]
@gelir95 Thank you for your feedback. I will take care of your suggestions. Please keep in mind the current draft state. I know that I need to read over everything again.

File: src/de/changelog.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Changelog
+
+| Version               | Datum   | Codename            |
+| -------               | ----    | --------            |
+| [v1.0](./v1.0 "v1.0") | XX-2024 | Decurion            |
+
+## Codename Schema
+
+Einheiten bei der Römischen Armee, siehe [hier](https://en.wikipedia.org/wiki/List_of_Roman_army_unit_types "Einheiten der römischen Armee") for reference.
```
Comment (DerZade) [inline src/de/changelog.md]: Nice touch! :)

I would have gone with something like star wars planets, but I'm fine with that as well :)

File: src/de/communities.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+![Überblick Einladungen](../images/communities/invites-overview.webp "Überblick Einladungen")
+
+Spieler können deiner Community nur mit einer gültigen Einladung betreten. Eine Einladung ist ein Link, kombiniert mit einem Ablaufdatum und einer Höchstanzahl an Verwendungen. Standardmäßig ist eine Einladung für 7 Tage gültig und unbegrenzt nutzbar. Du kannst diese Einstellungen ändern durch klicken auf **Einladungs-Link bearbeiten** in dem Fenster das erscheint wenn du auf **+ Einladung erstellen** klickst.
+
+![Einladungs-Links bearbeiten](../images/communities/edit-invite-links.webp "Einladungs-Links bearbeiten")
+
+Die verbleibende Zeit und die Anzahl der Verwendungen wird in der Einladungsübersicht angezeigt. Über das Drei-Punkte-Symbol kannst du das plattformabhängige **Teilen**-Menü öffnen, eine aktive Einladung **löschen** oder **Link kopieren** benutzen. Außerdem kannst du den Link schnell kopieren indem du auf den Einladungs-Code klickst.
+
+Sende nun den erstellten Link über deine bevorzugte Methode an deine Community-Mitglieder. Das Format eines Einladungslinks ist stets `https://arma.events/[EINLADUNGS-CODE]` also z.B. `https://arma.events/1YOtETcO`
```
Comment (DerZade) [inline src/de/communities.md]:
```suggestion
Sende nun den erstellten Link über deine bevorzugte Methode an deine Community-Mitglieder. Das Format eines Einladungslinks ist stets `https://arma.events/i/[EINLADUNGS-CODE]` also z.B. `https://arma.events/i/1YOtETcO`
```

File: src/de/communities.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+### Mitglieder
+
+Im Mitglieder-Tab kannst du nach Mitgliedern in der Mitgliederliste suchen (1), einem Mitglied Admin-Rechte erteilen/entziehen oder ein Mitglied aus der Community entfernen (2). Nur Admins können Mitglieder oder andere Admins entfernen. Nur Admins können Mitglieder zu Admins befördern. Wenn du eine Community erstellst, wirst du automatisch der erste Admin. Jeder Admin hat ein <img src="../images/general/crown-text-color-2.svg" style="display:inline" width="16" height="16" alt="Krone"> Kronensymbol direkt neben seinem Namen. Siehe [Rollen](#rollen) für weitere Informationen.
+
+### Rollen
+
+| Rolle       | Beschreibung |
+| -----       | ------------ |
+| Jedermann   | Standardrolle für alle **arma-events** Benutzer. Jeder kann deine Community- und Benutzer-Profilseiten sowie alle öffentlich einsehbaren Events sehen. |
+| Gast        | Gäste (einzelne Benutzer oder ganze Communities) können Events fremder Communities per Einladung Events beitreten. Muss in den Event-Einstellungen festgelegt werden. |
```
Comment (DerZade) [inline src/de/communities.md]: Gibt keinen Link für Event Einladungen. Die Event tauchen einfach unter "meine Events" auf der Startseite auf. Auf Dauer wär hier natürlich ein Workflow sinnvoll, dass nicht jede random Community einfach andere einladen kann ohne Absprache, aber ist aktuell noch nicht drin.

File: src/de/events.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+---
+outline: deep
+cSpell: locale de
+---
+
+# Events
+
+## Event erstellen
+
+Du kannst jederzeit ein Event für deine Community erstellen indem du das `+` Symbol neben deinem Avatar anklickst und **Event erstellen** auswählst.
```
Comment (DerZade) [inline src/de/events.md:9]: Vielleicht wär hier n Hint sinnvoll in dem nochmal klar gestellt wird, dass ein Event immer einer Community angehört. Das ist glaub ich nicht jedem sofort klar 🤔

File: src/en/communities.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
+---
+outline: deep
+cSpell: locale en
```
Comment (DerZade) [inline src/en/communities.md]: Macht es nicht sinn das über die vscode settings global einzustellen, anstatt bei jeder Datei ein `cSpell` drüber zu haben?

**Review by DerZade — COMMENTED** [review]
Currently some images are included 4 times (5 if you include the raw screenshot) in the repo. Is this really necessary? As far as I can tell only one is used.

The repo is already approaching 400MB. We should also try to reduce that before merging and then squash merge to get rid of old commits, which include all deleted objects.

File: src/de/communities.md (hunk 3 above)
Comment (y0014984) [reply src/de/communities.md]: Roger. Will change that in both languages.

File: src/de/communities.md (hunk 4 above)
Comment (y0014984) [reply src/de/communities.md]: Mit Einladung meinte ich jetzt keinen Link sondern den Vorgang "jemanden einladen" durch hinzufügen der Community in den Event-Einstellungen. Aber du hast Recht. Man sollte das an dieser Stelle präzisieren.

File: src/de/events.md (hunk 5 above)
Comment (y0014984) [reply src/de/events.md:9]: Ja, darüber hab ich auch schon nachgedacht. Werde ich nachreichen.

File: src/en/communities.md (hunk 6 above)
Comment (y0014984) [reply src/en/communities.md]: Aktuell unterscheide ich hier zwischen deutscher und englischer Rechtschreibprüfung auf Dokumentebene. Mir ist bisher kein Weg bekannt, wie ich dies global einstellen kann, sodass die deutschen Artikel nach deutscher und die englischen Artikel nach englischer Rechtschreibung geprüft werden (z.B. auf Ordnerebene). Ein ständiges hin- und herwechseln in den VSCode Settings, wenn ich an beiden Sprachen parallel arbeite, habe ich nicht als effektiv empfunden. Aber ich bin für Vorschläge offen.

**Comment by y0014984** [comment]
> Is this really necessary?

No, it's not. It's the result of a non-destructive multi-iterational process with a lost storage space overview. Thank you for keeping track of that. 

Of course, the commits will be merged as a squash merge. Now let's discuss which files it should contain.

| Files | Size | Necessary for production |
| ----- | ---- | ---------------------------- |
| raw screenshots | 15 MB and increasing | not, if already used |
| raw screen captures | 37 MB and increasing | not, if already used |
| AI generated | 62 MB and increasing | not, if already used |
| Davinci Resolve video projects (*.drp) | < 1 MB | yes |
| Videos as WEBM (used) | 1,5 MB | yes |
| Videos as GIF (not used anymore) | 210 MB | no |
| Affinity Designer image projects (*.afdesign) | 44 MB | yes |
| SVG Exports for editing in other software | 12 MB | only if someone else wants to modify my edit |
| Images as WEBP (used) | 5 MB | yes |
| Images as PNG (not used anymore) | 8 MB | no |

My suggestion is to delete the PNGs and the GIFs entirely. I can also keep the raw and AI files as well as the Affinity Designer projects locally (outside the repository) if you wish. This would reduce the repo size to a bare minimum. In combination with the squash merge this procedure should reduce the repo size to the max.

Additionally I will try to not create commits on the same files that frequently, especially if media files are involved.

File: .vitepress/config.mts [hunk 7]
```diff
@@ -1,28 +1,206 @@
...
+            items: [
+              { text: 'Überblick', link: '/de/overview' },
+              { text: 'Benutzer', link: '/de/users' },
+              { text: 'Communities', link: '/de/communities' },
+              {
+                text: 'Events', 
+                link: '/de/events', 
+                items: [
+                  { text: 'Markdown', link: '/de/markdown' },
+                  { text: 'Slotsliste', link: '/de/slotlist' }
```
Comment (DerZade) [inline .vitepress/config.mts]:
```suggestion
                  { text: 'Slotliste', link: '/de/slotlist' }
```

or

```suggestion
                  { text: 'Slotlist', link: '/de/slotlist' }
```

Comment (y0014984) [reply .vitepress/config.mts]: Upsi. Da macht man einmal ein `cSpell: disable` und schon haut es einem in die Fresse. :sob:

**Review by DerZade — APPROVED** [review]

**Pull request merged**